.github/workflows/pull-request-master-merged.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: PR To master merged
```

Comment:
These rules should apply to main and master. The master branch should be renamed to main at some point